docs/dev/guide/contract-verification.md

```diff
@@ -2,18 +2,19 @@
 
 ## What is contract verification?
 
-When you deploy a Smart Contract on a network, you’re essentially deploying the bytecode produced by the Solidity compiler. The original Solidity code is not retained in any way.
+When you deploy a smart contract on a network, you’re essentially deploying the bytecode produced by the Solidity compiler.
+
+Contract verification is ensuring the correctness, security, and safety of the smart contract implementations which allows smart contract developers to prove and publish the source code of the contracts deployed on-chain.
 
-Verifying a contract entails determining whether the on-chain bytecode generated from the Smart Contract source code complies. If it does, we can state that we have confirmed the bytecode's source code's integrity. Hence the name – verification.
```

Comment:
I guess this text was actually the answer to the 'What is contract verification'. You have replaced it with the text that states contract verification benefits but not about what is it. I would keep this paragraph as it was before